passenger-service/src/main/java/com/vlad/kuzhyr/passengerservice/web/controller/PassengerController.java

```diff
@@ -0,0 +1,79 @@
+package com.vlad.kuzhyr.passengerservice.web.controller;
+
+import com.vlad.kuzhyr.passengerservice.service.PassengerService;
+import com.vlad.kuzhyr.passengerservice.web.request.PassengerRequest;
+import com.vlad.kuzhyr.passengerservice.web.response.PassengerResponse;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springdoc.core.annotations.ParameterObject;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/v1/passenger")
```

Comment:
```suggestion
@RequestMapping("/api/v1/passengers")
```
обычно при нейминге эндпоинтов используют множественное число